library/general/src/modules/Report.rb

```diff
@@ -39,6 +39,8 @@ module Yast
   # @TODO not all methods respect all environment, feel free to open issue with
   #   method that doesn't respect it.
   class ReportClass < Module
+    include Logger
```

Comment:
NP: I know this is correct, but I always prefer explicit `Yast::Logger` to not confuse it with the `Logger` from the Ruby stdlib.